eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -210,6 +210,25 @@ jobs:
     displayName: Publish Code Coverage to DevOps
     condition: eq(variables['AZ_SDK_CODE_COV'], 1)
 
+  - task: mspremier.BuildQualityChecks.QualityChecks-task.BuildQualityChecks@6
+    displayName: Check line coverage
+    inputs:
+      checkCoverage: true
+      coverageFailOption: fixed
```

Comment:
What other kinds of options are available here and what does `fixed` mean?

Reply:
`fixed` refers to `Fixed Threshold` which compares coverage against an absolute number. The alternative is `build` which compares coverage to a previous build (how they select the previous build is not documented) 

https://github.com/MicrosoftPremier/VstsExtensions/blob/master/BuildQualityChecks/en-US/CodeCoveragePolicy.md 

The other option is `build` which refers to coverage from a previous build. 

If we ever do a big refactor it might be worth switching to `useUncoveredElements`